### PR TITLE
Reverting all changes related to #1132 as this is breaking multiple things

### PR DESCRIPTION
**Pull request by jyotisingh**

### CODE REVIEW

File: base/src/com/thoughtworks/go/util/FileUtil.java [hunk 1]
```diff
@@ -64,21 +76,18 @@ public static String makepath(String... paths) {
...
-            return builder.toString().trim();
-        } finally {
-            IOUtils.closeQuietly(input);
+        @SuppressWarnings("unchecked") List<String> list = IOUtils.readLines(input);
+        StringBuilder builder = new StringBuilder();
+        for (String line : list) {
+            builder.append(line);
+            builder.append(lineSeparator());
         }
+        return builder.toString().trim();
```
Comment (ketan) [inline base/src/com/thoughtworks/go/util/FileUtil.java]: Can we just add a close quietly in this.method for good measure?

Comment (jyotisingh) [inline base/src/com/thoughtworks/go/util/FileUtil.java]: `IOUtils.closeQuietly` looks harmless, but so did the original PR :laughing: . The intent was to leave it in the form it was earlier. I can still add it if you think its necessary?
Weren't you off to sleep?

**Comment by arvindsv** [comment]
Seems ok to me.

```
$ git diff HEAD 15.2.0 -- base/src/com/thoughtworks/go/util/FileUtil.java
diff --git a/base/src/com/thoughtworks/go/util/FileUtil.java b/base/src/com/thoughtworks/go/util/FileUtil.java
index 02d3b08..6b84cf0 100644
--- a/base/src/com/thoughtworks/go/util/FileUtil.java
+++ b/base/src/com/thoughtworks/go/util/FileUtil.java
@@ -1,5 +1,6 @@
-/*
- * Copyright 2015 ThoughtWorks, Inc. *
+/*************************GO-LICENSE-START*********************************
+ * Copyright 2014 ThoughtWorks, Inc.
+ *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
@@ -11,7 +12,7 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
+ *************************GO-LICENSE-END***********************************/

 package com.thoughtworks.go.util;

$ git diff HEAD 15.2.0 -- common/test-resources/unit/data/MinifiedJunitReport.xml
$ git diff HEAD 15.2.0 -- common/test/unit/com/thoughtworks/go/domain/UnitTestReportGeneratorTest.java
$ git diff HEAD 15.2.0 -- common/test/unit/com/thoughtworks/go/util/FileUtilTest.java
$ git diff HEAD 15.2.0 -- config/config-api/src/com/thoughtworks/go/domain/UnitTestReportGenerator.java
```

File: base/src/com/thoughtworks/go/util/FileUtil.java (hunk 1 above)
Comment (arvindsv) [inline base/src/com/thoughtworks/go/util/FileUtil.java]: My usual thought about closing streams is: Whoever opens it closes it. At the same "level" (or method) that it is opened.

Comment (arvindsv) [inline base/src/com/thoughtworks/go/util/FileUtil.java]: But I see there are some incorrect usages of readToEnd, like in LoggingConfUpdateTransformer.

**Comment by jyotisingh** [comment]
waiting for someone to click on the merge button if it looks about ok? :smile:

**Pull request merged**